_posts/2016-10-02-distributed-array-of-commodity-hosts.md

```diff
@@ -0,0 +1,163 @@
+---
+layout: post
+title:  "Distributed Array of Commodity Hosts (DACH)"
```

Comment:
s/Hosts/Hosting Services/
s/DACH/DACHS/

We needed to pull the "S" over from the "SHUND" suffix, but as a first class component of the acronym; so we backronymed it into `Distributed Array of Commodity Hosting Services` ... :-)